src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(health, name) {
```

Comment:
The constructor of the Animal class should not accept a 'health' parameter since the health is always initialized to 100 by default according to the task description.